dash/backend/src/modules/falco/dao/falco.dao.ts

```diff
@@ -69,10 +69,10 @@ export class FalcoDao {
                     );
                     break;
                 case 'Date Desc':
-                    query = query.orderBy([{column: 'calendar_date', order: 'desc'}]);
+                    query = query.orderBy([{column: 'creation_timestamp', order: 'desc'}]);
```

Comment:
One thing to check - how does this query perform? Does it hit indexes or anything?